apps/web/src/routes/NonAuthLayoutRouter/Login/Login.js

```diff
@@ -21,6 +20,22 @@ export default function Login (props) {
   const { t } = useTranslation()
   const DEFAULT_LOGIN_ERROR = t('Sorry, that Email and Password combination didn\'t work.')
 
+  useEffect(() => {
+    const checkAutofill = () => {
+      const emailInput = document.getElementById('email')
+      const passwordInput = document.getElementById('password')
+      if (emailInput?.value && !email) {
+        setEmail(emailInput.value)
```

Comment:
i dont think ive seen issues with this not working before, why is this needed?

Reply:
You use mostly light mode you may not have experienced the issue. Maybe it's not a big deal to have off-color inputs, but it was bothering me. I can remove it and use default styles, it just looks a bit weird in dark mode.

Reply:
but what is this code doing? its setting the email to the input value, that part was not working? this has nothing to do with styling.